Source/SwiftLintFramework/Rules/OperatorUsageWhitespaceRule.swift

```diff
@@ -105,7 +107,7 @@ public struct OperatorUsageWhitespaceRule: OptInRule, CorrectableRule, Configura
             zeroSpaces + trailingVariableOrNumber
         let excludingPattern = "(?:\(genericPattern)|\(validRangePattern))"
 
-        let kinds = SyntaxKind.commentAndStringKinds()
+        let kinds = SyntaxKind.commentAndStringKinds() + [.objectLiteral]
```

Comment:
I would suspect that other rules that exclude comments and strings would also need to exclude object literals.

This is why checking for explicit matching kinds is always preferable to excluding.